dev/devicelab/bin/tasks/module_test_ios.dart

```diff
@@ -703,13 +703,7 @@ Future<bool> _isAppAotBuild(Directory app) async {
     'App',
   );
 
-  final String symbolTable = await eval(
-    'nm',
-    <String> [
-      '-gU',
```

Comment:
`-U`'s description: `Don't display undefined symbols.` This isn't strictly necessary, it reduces the output.